src/js/events/ChronosActions.js

```diff
@@ -40,10 +49,102 @@ const ChronosActions = {
     },
     {delayAfterCount: Config.delayAfterErrorCount}
   ),
+
+  fetchJobDetail: function (jobID) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/chronos/jobs/${jobID}`,
+      data: [{name: 'embed', value: 'active-runs'}],
```

Comment:
Yeah, this is the data that the view needs to display. See the design doc for the API here: https://docs.google.com/document/d/1xYlGTs5FTD4CBNICSjmAaD6SDSekL_-iLQNljNdCkjI/edit?ts=574db9f3#heading=h.2zxnl8tuxax6